javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,146 @@
+/*freecodecamp.org account name: mehmetnuricavdar
+https://www.freecodecamp.org/mehmetnuricavdar
+*/
+
+// ######    Flight booking fullname function
+
+function getFullname(firstname, surname) {
+  const fullName = `${firstname} ${surname}`;
+  return fullName;
+}
+
+let fullname1 = getFullname("Benjamin", "Hughes");
+let fullname2 = getFullname("Mehmet", "Cavdar");
+
+// ###### Formal fullname
+
+function getFullname(firstname, surname, useFormalName, gender) {
+  const fullName = `${firstname} ${surname}`;
+  if (useFormalName === true) {
+    if (gender === "male") {
+      return `Lord ${fullName}`;
+    } else {
+      return `Lady ${fullName}`;
+    }
+  } else {
+    return fullName;
+  }
+}
+getFullname("Benjamin", "Hughes", true, "male"); // returns "Lord Benjamin Hughes"
+getFullname("Benjamin", "Hughes", false); // returns "Benjamin Hughes"
+getFullname("Benjamin", "Hughes", true, "female"); // returns "Lady Benjamin Hughes"
+
+// #####  Event application
+
+function getEventWeekday(futureDays) {
+  const week = [
+    "Sunday",
+    "Monday",
+    "Tuesday",
+    "Wednesday",
+    "Thursday",
+    "Friday",
+    "Saturday",
+  ];
+  let today = new Date().getDate();
+  const eventDate = (today + futureDays) % week.length;
+  return week[eventDate];
+}
+
```

Comment:
This solution doesn't work for me, I tested it today, and with the argument of 2, it gave me Tuesday. You need to make some changes to your eventDate, and maybe use `getDay` method

Reply:
ahh yes i guess after "Da" i did auto complete and it completed "Date" instead of "Day"